subsys/bluetooth/controller/ll_sw/ull_slave.c

```diff
@@ -213,6 +213,21 @@ void ull_slave_setup(memq_link_t *link, struct node_rx_hdr *rx,
 		}
 	}
 
+#if defined(CONFIG_BT_CTLR_ADV_EXT)
+	if (adv->is_created & BIT(1)) {
+		ll_rx_put(link, rx);
+
+		/* use reserved link and node_rx to prepare advertising terminate event */
+		rx = adv->node_rx_adv_term;
+		link = rx->link;
+		rx->handle = ull_adv_handle_get(adv);
+		rx->type = NODE_RX_TYPE_EXT_ADV_TERMINATE;
+		rx->rx_ftr.param = (void *)((u32_t)lll->handle);
+		rx->rx_ftr.extra = (void *)(adv->lll.numAdvEventsSent);
```

Comment:
set this to 0. and have a TODO comment to mention missing implementation of Max_Extended_Advertising_Events parameter and duration parameter in HCI_LE_Set_Extended_Advertising_Enable.